### PR TITLE
Adds Virtual League for all verified accounts

### PR DESCRIPTION
**Pull request by johnkors**
Screenshot 2021-02-26 at 10 28 36

Screenshot 2021-02-26 at 10 28 55

### CODE REVIEW

File: pages/verified/[...slug].tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+  if (query.slug) {    
+    const entryId = parseInt(query.slug[0]);
+
+    if (!entryId)
+      return { verifiedEntryData: { type: "NO_SLUG" } }
+    
+    const verifiedEntry = await getVerifiedEntry(entryId);
+    
+    return {
```
Comment (kristianeaw) [inline pages/verified/[...slug].tsx:167]: return her burde være `return { verifiedEntryData: verifiedEntry }`

File: pages/virtual-leagues/all.tsx/index.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import VerifiedPage from "../../../components/virtual-leagues/VerifiedPage";
```
Comment (kristianeaw) [inline pages/virtual-leagues/all.tsx/index.tsx:1]: mappa burde vel ikke hete .tsx 😬

Comment (johnkors) [reply pages/virtual-leagues/all.tsx/index.tsx:1]: ugg, takk. Miss

File: pages/virtual-leagues/all/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import VerifiedPage from "../../../components/virtual-leagues/VerifiedPage";
+import { getVerifiedEntries, GetVerifiedEntriesResponse } from "../../../services/verified";
+
+const Page = (verifiedEntriesResponse:GetVerifiedEntriesResponse) => {
+  return(<>
```
Comment (kristianeaw) [inline pages/virtual-leagues/all/index.tsx:5]: pirk: her kunne du hatt 2 returns i stedet for ternary

**Review by kristianeaw — APPROVED** [review]

**Pull request merged**